harness/determined/pytorch/__init__.py

```diff
@@ -25,5 +25,16 @@
 )
 from determined.pytorch._experimental import PyTorchExperimentalContext
 from determined.pytorch._pytorch_context import PyTorchTrialContext
-from determined.pytorch._pytorch_trial import PyTorchTrial, PyTorchTrialController
+from determined.pytorch._pytorch_trial import (
+    PyTorchTrial,
+    PyTorchTrialController,
+    TrainUnit,
+    # XXX: do these belong here? should only be used in tests
```

Comment:
Yeah that's fine.  This is [covered explicitly](https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md#special-case-handling-private-functions-and-classes-in-private-modules) in the code layout rfc.